frontend/src/metabase/ui/components/theme/DatesProvider/DatesProvider.tsx

```diff
@@ -1,7 +1,7 @@
 import type { ReactNode } from "react";
-import dayjs from "dayjs";
 import { DatesProvider as MantineDatesProvider } from "@mantine/dates";
 import type { DatesProviderSettings, DayOfWeek } from "@mantine/dates";
+import dayjs from "metabase/lib/dayjs";
 
```

Comment:
@ranquild dayjs throws an error preventing storybook from working at all for me locally because we're using an unextended dayjs here, but relying on a method from one of the extensions.

![Screen Shot 2023-10-20 at 1 10 05 PM](https://github.com/metabase/metabase/assets/30528226/6a689020-d490-4d7d-bbd4-d515fee90080)

It works in the main app and jest because we import this in app.js and jest-setup.js

Let me know if you think there's a better pattern than just exporting the extended object from "metabase/lib/dayjs";

Reply:
nevermind, looks like this got fixed since I branched off master